src/main/webapp/src/app/nav-bar/nav-bar.component.ts

```diff
@@ -1,19 +1,13 @@
 import { Component, OnInit, Output, EventEmitter } from '@angular/core';
```

Comment:
remove output/eventemitter if no longer in use

Reply:
removed